djstripe/stripe_objects.py

```diff
@@ -139,26 +136,60 @@ def __str__(self):
 
 
 class StripeEvent(StripeObject):
+    """
+    Events are POSTed to our webhook url. They provide information about a Stripe event that just happened. Events
+    are processed in detail by their respective models (charge events by the Charge model, etc).
+
+    Events are initially _UNTRUSTED_, as it is possible for any web entity to post any data to our webhook url. Data
+    posted may be valid Stripe information, garbage, or even malicious. The 'valid' flag in this model monitors this.
+
+    API VERSIONING
+    ====
+    This is a tricky matter when it comes to webhooks. See the discussion here:
+        https://groups.google.com/a/lists.stripe.com/forum/#!topic/api-discuss/h5Y6gzNBZp8
+
+    In this discussion, it is noted that Webhooks are produced in one API version, which will usually be
+    different from the version supported by Stripe plugins (such as djstripe). The solution, described there,
+    is that the receipt of a webhook event should be 1st) validated by doing an event get using the API version
```

Comment:
Go ahead and display the 3 steps in the form of a list. Makes it stand out a bit better.